app/src/interfaces/file/file.vue

```diff
@@ -89,6 +102,24 @@ const edits = computed(() => {
 	return props.value;
 });
 
+const values = inject('values', ref<Record<string, any>>({}));
+
+const customFilter = computed(() => {
+	return parseFilter(
+		deepMap(props.filter, (val: any) => {
+			if (val && typeof val === 'string') {
+				return render(val, values.value);
+			}
+
+			return val;
+		}),
+	);
+});
+
+const _disabled = computed(() => {
```

Comment:
```suggestion
const disabled = computed(() => {
```

Lets rename `_disabled` to `disabled`. Variable names starting with underscore imply that it is unused in typescript.